public/app/plugins/datasource/loki/modifyQuery.ts

```diff
@@ -154,7 +155,7 @@ export function addLabelToQuery(
   key: string,
   operator: string,
   value: string,
-  forceAsLabelFilter = false
+  labelType?: LabelType | null
```

Comment:
function documentation still has this as `forceAsLabelFilter`
```
* @param {boolean} [forceAsLabelFilter=false] - if true, it will add a LabelFilter expression even if there is no parser in the query
```

Reply:
Classic issue with jsdoc comments vs using TypeScript as documentation.

Reply:
removed all params in b706e425ddb01cace41ca939b558f1aa3107d8a4